src/SignalR/clients/csharp/Client.SourceGenerator/src/GeneratorHelpers.cs

```diff
@@ -25,4 +25,13 @@ public static string GetAccessibilityString(Accessibility accessibility)
                 return null;
         }
     }
+
+    public static string SourceFilePrefix()
+    {
+        return @"// <auto-generated>
+// Generated by Microsoft.AspNetCore.Client.SourceGenerator
```

Comment:
What do you think about adding the assembly file version to the header? e.g.

> Generated by Microsoft.AspNetCore.Client.SourceGenerator 7.0.22.35609

Makes it easier to see at a glance what version of the generator ran.

Reply:
This is what comes from the AssemblyInfo generator:

```
//------------------------------------------------------------------------------
// <auto-generated>
//     This code was generated by a tool.
//     Runtime Version:4.0.30319.42000
//
//     Changes to this file may cause incorrect behavior and will be lost if
//     the code is regenerated.
// </auto-generated>
//------------------------------------------------------------------------------
```

Reply:
Sounds like a good addition once we start shipping this package.